release/_package.sh

```diff
@@ -92,6 +92,7 @@ function package_boms {
 	lc_cd "${_BUILD_DIR}/boms"
 
 	cp -a ./*.pom "${_BUILD_DIR}/release"
+	cp -a ./*.jar "${_BUILD_DIR}/release"
```

Comment:
@pgydev, we need to copy only the distro jar, am I right?

If so, can you please adjust to the following code and resend the pull?
```
cp release."${LIFERAY_RELEASE_PRODUCT_NAME}.distro-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}".jar "${_BUILD_DIR}/release"
```
I'm concerned of we copy any unintended jar file :slightly_smiling_face: